stdlib/public/core/StringCharacterView.swift

```diff
@@ -107,10 +142,27 @@ extension String: BidirectionalCollection {
   ///   to `index(before:)`.
   ///
   /// - Complexity: O(*n*), where *n* is the absolute value of `n`.
-  @inlinable @inline(__always)
-  public func index(_ i: Index, offsetBy n: IndexDistance) -> Index {
+  public func index(_ i: Index, offsetBy distance: Int) -> Index {
```

Comment:
Err, we need/should document if we drop inlinable

Reply:
Yep, there is a comment immediately below.